src/marked/marked.module.scss

```diff
@@ -288,7 +290,7 @@
   }
 
   pre {
-    background-color: #f6f8fa;
+    background-color: var(--terra-dev-site-marked-pre-background-color, #f6f8fa);
```

Comment:
We should keep a consistent scoped prefix. This variable is correct, the variables above may need updated.

This prefix makes sense to me:

`--terra-dev-site-marked`